ui/app/AppLayouts/Wallet/views/TransactionSettings.qml

```diff
@@ -106,7 +107,13 @@ Rectangle {
             const priorityFeeWei = Utils.gweiToWei(customPriorityFeeInput.text)
             const totalFee = SQUtils.AmountsArithmetic.sum(baseFeeWei, priorityFeeWei)
             const feeInWei = SQUtils.AmountsArithmetic.times(totalFee, SQUtils.AmountsArithmetic.fromString(customGasAmountInput.text)).toFixed()
+            const estimatedTime = root.fnGetEstimatedTime(totalFee.toFixed(), priorityFeeWei.toFixed())
             optionCustom.subText = root.fnGetPriceInCurrencyForFee(feeInWei)
+            if (estimatedTime === 61) {
```

Comment:
```suggestion
            if (estimatedTime > 60) {
```

?

Reply:
@caybro updated. Created `formatEstimatedTime` function to format the time.